### DISCUSSION TITLE
响应

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1328] [General Question] [answered]
老师，我想问一下在项目当中响应是在哪里处理的（我们从api接口处获取到的响应内容不止回复的文本，还有别的东西比如token数），我们想要在响应输出给用户的时候加上token数，在哪里修改可以实现

**Comment by arvinxx** [comment]
目前是直接转成纯文本到浏览器了。要加token数比较麻烦哎

**Comment by 5478-lfy** [comment]
@arvinxx 老师，那这个响应的位置在哪里，我这边没有找到，嘿嘿，拜托

**Reply by arvinxx** [reply, marked as answer]
https://github.com/lobehub/lobe-chat/blob/main/src/app/api/chat/%5Bprovider%5D/route.ts 从这个入口往下找

**Comment by 5478-lfy** [comment]
好的谢谢
